reactive/apache.py

```diff
@@ -45,7 +46,10 @@ def config_changed():
 def install_packages(workload):
     config = hookenv.config()
     hookenv.status_set('maintenance', 'Installing packages')
-    packages = ['apache2', 'php5-cgi', 'libapache2-mod-php5']
+    if lsb_release()['DISTRIB_CODENAME'].lower() >= 'xenial':
```

Comment:
The '>=' operator on the string looks funny. Will this code keep working when the distribution names cycle?